crates/bevy_reflect/Cargo.toml

```diff
@@ -39,6 +39,8 @@ glam = { version = "0.28", features = ["serde"], optional = true }
 petgraph = { version = "0.6", features = ["serde-1"], optional = true }
 smol_str = { version = "0.2.0", optional = true }
 uuid = { version = "1.0", optional = true, features = ["v4", "serde"] }
+inventory = "0.3"
+wasm-init = "0.2"
```

Comment:
These could be feature gated behind the targets that respectively use them, in order to avoid compiling them when not necessary.

Reply:
Yeah, I agree.

Reply:
This seems resolved in the latest commits so you can mark the comment as such